pom.xml

```diff
@@ -37,13 +37,27 @@
 			<groupId>net.imglib2</groupId>
 			<artifactId>imglib2-realtransform</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>net.imglib2</groupId>
+			<artifactId>imglib2-algorithm-gpl</artifactId>
```

Comment:
This is a problem. The `imglib2-algorithm-gpl` component must not be a dependency here, both for legal reasons, as well as logistical/organizational ones.